src/panels/config/integrations/ha-config-integrations.ts

```diff
@@ -100,6 +112,69 @@ class HaConfigIntegrations extends SubscribeMixin(LitElement) {
     ];
   }
 
+  private _filterConfigEntries = memoizeOne(
+    (configEntries: ConfigEntry[], filter?: string): ConfigEntry[] => {
+      if (filter) {
+        let configEntriesSearch: ConfigEntry[] = configEntries.map(
+          (configEntry: ConfigEntry) => {
+            return {
+              ...configEntry,
+              domain: domainToName(this.hass.localize, configEntry.domain),
+            };
+          }
+        );
+        const options: Fuse.FuseOptions<ConfigEntry> = {
+          keys: ["domain", "title"],
+          caseSensitive: false,
+          minMatchCharLength: 2,
+          threshold: 0.2,
+        };
+        const fuse = new Fuse(configEntriesSearch, options);
+        configEntriesSearch = fuse.search(filter);
+        configEntries = configEntries.filter(
+          (configEntry: ConfigEntry) =>
+            configEntriesSearch.findIndex(
+              (ce: ConfigEntry) => ce.entry_id === configEntry.entry_id
+            ) !== -1
+        );
+      }
+      return configEntries;
+    }
+  );
+
+  private _filterConfigEntriesInProgress = memoizeOne(
+    (
+      configEntriesInProgress: DataEntryFlowProgress[],
+      filter?: string
+    ): DataEntryFlowProgress[] => {
+      if (filter) {
+        let configEntriedInProgressSearch: DataEntryFlowProgressSearch[] = configEntriesInProgress.map(
```

Comment:
Same here, add it in the subscribe function. Make a new extended type.

Reply:
Struggling here to get translations to load after subscribe change level (line 107). Its an async function is called but even if I try to move around and await that call (this.hass.loadBackendTranslation) it still returns blank, like the translation hasn't loaded yet.

The only way I've been able to get the translation is to load it at render or filter level like it is now.. :confused: 

Any ideas?